check_addon.py

```diff
@@ -302,11 +302,12 @@ def _check_file_whitelist(error_counter, file_index, addon_path):
 
 def _logProblem(error_counter, problem_string):
     colorPrint("PROBLEM: %s" % problem_string, "31")
+    comments.append("PROBLEM: %s" % problem_string)  
```

Comment:
Might be nice have a combined api for your comments and the colorPrint interface. As the message should always be identical.